stdlib/asyncio/base_events.pyi

```diff
@@ -86,7 +98,15 @@ class BaseEventLoop(AbstractEventLoop):
     # Future methods
     def create_future(self) -> Future[Any]: ...
     # Tasks methods
-    if sys.version_info >= (3, 8):
+    if sys.version_info >= (3, 11):
+        def create_task(
+            self,
+            coro: Coroutine[Any, Any, _T] | Generator[Any, None, _T],
```

Comment:
Should we create an alias somewhere (`_typeshed`?) for `Coroutine[Any, Any, _T] | Generator[Any, None, _T]`? I think we could use it quite a lot in our `asyncio` stubs. (No need to do that in this PR.)

Reply:
It's probably worth checking whether we still need the Generator variant. That feels like a holdover from the pre-`await` days.